addons/popochiu/engine/audio_manager/audio_cue.gd

```diff
@@ -93,9 +93,12 @@ func set_loop(value: bool) -> void:
 		'AudioStreamOggVorbis', 'AudioStreamMP3':
 			audio.loop = value
 		'AudioStreamWAV':
-			(audio as AudioStreamWAV).loop_mode =\
-			AudioStreamWAV.LOOP_FORWARD if value\
-			else AudioStreamWAV.LOOP_DISABLED
+			if (audio as AudioStreamWAV).get_loop_end() == 0 && value:
+				push_warning("res://addons/popochiu/engine/audio_manager/audio_cue.gd:97 " + resource_name + " does not have the correct metadata for loop, please check AudioStreamWAV documentation")
```

Comment:
I suggest using this way of displaying the warning so that it is understood as something triggered by the plugin and avoid long code lines:

```gdscript
PopochiuUtils.print_warning(
	"[b]%s[/b]" % resource_name +\
	" does not have the correct metadata to loop, please check" +\
	" AudioStreamWAV documentation"
)
```

Reply:
that's great, I will change it